VimeoNetworking/Sources/Models/VIMLiveChatUser.swift

```diff
@@ -47,12 +47,36 @@ public enum AccountType: String
 /// An object representing the `user` field in a `chat` response.
 public class VIMLiveChatUser: VIMModelObject
 {
+    /// Objecive-C accessible constant for the "Vimeo Basic" tier.
+    @objc public static let AccountTypeBasic = "basic"
+    
+    /// Objecive-C accessible constant for the "Vimeo Business" tier.
+    @objc public static let AccountTypeBusiness = "business"
+    
+    /// Objecive-C accessible constant for the "Business Live" tier.
+    @objc public static let AccountTypeLiveBusiness = "live_business"
+    
+    /// Objecive-C accessible constant for the "PRO Live" tier.
+    @objc public static let AccountTypeLivePro = "live_pro"
+    
+    /// Objecive-C accessible constant for the "Vimeo Plus" tier.
+    @objc public static let AccountTypePlus = "plus"
+    
+    /// Objecive-C accessible constant for the "Vimeo PRO" tier.
+    @objc public static let AccountTypePro = "pro"
+    
+    /// Objecive-C accessible constant for the "Custom Live" tier.
+    @objc public static let AccountTypeProUnlimited = "pro_unlimited"
+    
```

Comment:
Why are we adding these? Is this part of the conversion?

Reply:
Ah I think I had brought these in around the time I was working on the live chat stuff. We shouldn't need them anymore.